sdk/monitor/monitor-ingestion/README.md

```diff
@@ -0,0 +1,171 @@
+
+To authenticate, the following example uses `DefaultAzureCredential` from the [@azure/identity](https://www.npmjs.com/package/@azure/identity) package:
+
+```ts
+import { DefaultAzureCredential } from "@azure/identity";
+import { LogsIngestionClient } from "@azure/monitor-ingestion";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const logsIngestionEndpoint = process.env.LOGS_INGESTION_ENDPOINT || "logs_ingestion_endpoint";
+
+const credential = new DefaultAzureCredential();
+const logsIngestionClient = new LogsIngestionClient(logsIngestionEndpoint, credential);
+```
+
+## Key concepts
+
+### Data Collection Endpoint
+
+Data Collection Endpoints (DCEs) allow you to uniquely configure ingestion settings for Azure Monitor. [This 
+article][data_collection_endpoint] provides an overview of data collection endpoints including their contents and 
+structure and how you can create and work with them.
+
+### Data Collection Rule
+
+Data collection rules (DCR) define data collected by Azure Monitor and specify how and where that data should be sent or
+stored. The REST API call must specify a DCR to use. A single DCE can support multiple DCRs, so you can specify a
+different DCR for different sources and target tables.
+
+The DCR must understand the structure of the input data and the structure of the target table. If the two don't match,
+it can use a transformation to convert the source data to match the target table. You may also use the transform to
+filter source data and perform any other calculations or conversions.
+
+For more details, refer to [Data collection rules in Azure Monitor][data_collection_rule].
+
+### Log Analytics workspace tables
+
+Custom logs can send data to any custom table that you create and to certain built-in tables in your Log Analytics 
+workspace. The target table must exist before you can send data to it. The following built-in tables are currently supported:
+
+- [CommonSecurityLog](https://docs.microsoft.com/azure/azure-monitor/reference/tables/commonsecuritylog)
+- [SecurityEvents](https://docs.microsoft.com/azure/azure-monitor/reference/tables/securityevent)
+- [Syslog](https://docs.microsoft.com/azure/azure-monitor/reference/tables/syslog)
+- [WindowsEvents](https://docs.microsoft.com/azure/azure-monitor/reference/tables/windowsevent)
+
+## Examples
+
+- [Upload custom logs](#upload-custom-logs)
+- [Verify logs](#verify-logs)
+
+You can familiarize yourself with different APIs using [Samples](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/monitor/Azure.Monitor.Ingestion/samples).
+
+### Upload custom logs
+
+You can create a client and call the client's `Upload` method. Take note of the data ingestion [limits](https://docs.microsoft.com/azure/azure-monitor/service-limits#custom-logs).
+
+
+```ts
+import { DefaultAzureCredential } from "@azure/identity";
+import { LogsIngestionClient } from "@azure/monitor-ingestion";
+
+import * as dotenv from "dotenv";
```

Comment:
I think we should present examples in JS. Should we use cjs syntax for now until we figure out our migration path to the JS's module syntax?

Reply:
ooh I have been providing TS samples everywhere in the readme

Reply:
There are *some* cases where we give TS examples, but it's when there are TS-specific things like leveraging generics: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/search/search-documents/README.md#querying-with-typescript

Otherwise, yeah, stick to JS so folks can copy-paste into a node terminal interactively :)

Reply:
Switched readme samples to JS